README.md

```diff
@@ -48,33 +48,63 @@ Note, to use the `load` method on Blocks, you must already have a block document
 
 ### Write and run a flow
 
-Create table named customers and insert values.
+#### Using a SyncDriver with DatabaseCredentials
+Use `DatabaseCredentials` as a context manager to `execute` and `execute_many` operations; then, `fetch_many` and `fetch_one` operations.
+
 ```python
-from prefect import flow
+from prefect_sqlalchemy import DatabaseCredentials, SyncDriver
+
+with DatabaseCredentials(
+    driver=SyncDriver.SQLITE_PYSQLITE,
+    database="my.db",
+) as database_credentials:
+    database_credentials.execute(
+        "CREATE TABLE IF NOT EXISTS customers (name varchar, address varchar);"
+    )
+    database_credentials.execute(
+        "INSERT INTO customers (name, address) VALUES (:name, :address);",
+        parameters={"name": "Marvin", "address": "Highway 42"},
+    )
+    database_credentials.execute_many(
+        "INSERT INTO customers (name, address) VALUES (:name, :address);",
+        seq_of_parameters=[
+            {"name": "Ford", "address": "Highway 42"},
+            {"name": "Unknown", "address": "Highway 42"},
+        ],
+    )
+    # Repeated fetch* calls using the same operation will skip re-executing and instead return the next set of results
```

Comment:
is the asterisk intended?

Reply:
Yep referencing fetch_one, fetch_many, fetch_all. Perhaps I can make it more explicit by adding the underscore